### PR TITLE
extended HCP policies that CU can opt-in

### PR DESCRIPTION
**Pull request by lnguyen1401**
### What type of PR is this?
_(bug/feature/cleanup/documentation)_
documentation
### What this PR does / why we need it?
A list of permissions in addition to the current hcp support role that cu can opt in to allow SRE more access to troubleshoot
### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/OSD-19495
_Fixes #_

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
@lnguyen1401 Does this includes the perms to run network verifer?

**Comment by lnguyen1401** [comment]
@wanghaoran1988 Yes it does :
                               "ec2:RunInstances",
				"ec2:DescribeInstanceTypes",
				"ec2:CreateTags",
				"ec2:GetConsoleOutput"

File: resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                "elasticloadbalancing:ConfigureHealthCheck",
+                "kms:CreateGrant",
+                "s3:GetBucketTagging",
+                "s3:GetObjectAcl",
+                "s3:GetObjectTagging",
+                "s3:ListAllMyBuckets",
+                "tiros:CreateQuery",
+                "tiros:GetQueryAnswer",
+                "tiros:GetQueryExplanation",
+                "ec2:RunInstances",
```
Comment (wanghaoran1988) [inline resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json:97]: I am not seeing the "TerminateInstances" perms to run the osd-network-verifier, are you removing that on purpose?

Comment (wanghaoran1988) [reply resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json:97]: Ignore above, I see it's already in the default support policy.

**Pull request merged**